### PR TITLE
Converting documentation files to readthedocs format

### PR DESCRIPTION
**Pull request by giovannicuriel**
This PR reformats the documentation for DeviceManager to be compatible with readthedocs format (rst). 

This change was needed because versioning documentation in readthedocs is much easier and well integrated than in GitHub pages (and now there's a sphinx extension to render API blueprints).

### CODE REVIEW

File: docs/concepts.rst [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+sensors/attributes inferred from other devices.
+
+Consider, for instance, an actual device with temperature and humidity
+sensors; it can be represented into dojot as a device with two
+attributes (one for each sensor). We call this kind of device as regular
+device or by its communication protocol, for instance, MQTT device or
+CoAP device.
+
+We can also create devices which don’t directly correspond to their
+actual ones, for instance, we can create one with higher level of
```
Comment (ericbaum) [inline docs/concepts.rst]: > actual ones

physical versions

File: docs/concepts.rst [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+   -  type (string, read-write, required): attribute type (“static” or
+      “dynamic”)
+   -  value_type (string, read-write, required): attribute value type
+      (“string”, “float”, “integer”, “geo”)
+   -  static_value (string, read-write): if this is a static attribute,
+      which is its static value
+   -  template_id (string, read-write): from which template did this
+      attribute come from.
+
+\*although the code allows the user to set this.
```
Comment (ericbaum) [inline docs/concepts.rst]: Is this phrase floating?

File: docs/conf.py [hunk 3]
```diff
@@ -0,0 +1,183 @@
...
+#
+# source_suffix = ['.rst', '.md']
+source_suffix = '.rst'
+
+# The master toctree document.
+master_doc = 'index'
+
+# General information about the project.
+project = u'DeviceManager'
+copyright = u'2017, CPqD'
```
Comment (ericbaum) [inline docs/conf.py]: 2018?

File: docs/conf.py [hunk 4]
```diff
@@ -0,0 +1,183 @@
...
+author = u'Matheus Magalhaes'
```
Comment (ericbaum) [inline docs/conf.py]: Giovanni Curiel

File: docs/concepts.rst (hunk 1 above)
Comment (giovannicuriel) [reply docs/concepts.rst]: Ok.

File: docs/concepts.rst (hunk 2 above)
Comment (giovannicuriel) [reply docs/concepts.rst]: Yes - removed.

File: docs/conf.py (hunk 4 above)
Comment (giovannicuriel) [reply docs/conf.py]: Appended :)

**Pull request merged**